plugins/functions/string.c

```diff
@@ -21,8 +21,18 @@
 #  define GRN_PLUGIN_FUNCTION_TAG functions_string
 #endif
 
+#ifdef HAVE_CONFIG_H
+//Ambiguous (onigmo has the same name header).
+# include <config.h>
+#endif /* HAVE_CONFIG_H */
+
```

Comment:
Do we need to include `config.h` explicitly?
Can we include `grn_onigmo.h` without this?

Reply:
Could you confirm this?